spec/presenters/export/csv/project_presenter_spec.rb

```diff
@@ -480,6 +480,16 @@
     expect(presenter.advisory_board_conditions).to eql("These are the conditions.")
   end
 
+  it "presents the completed grant payment certificate received" do
+    mock_successful_api_response_to_create_any_project
+    tasks_data = build(:conversion_tasks_data, receive_grant_payment_certificate_date_received: Date.new(2024, 1, 1))
+    project = create(:conversion_project, tasks_data: tasks_data)
+
+    presenter = described_class.new(project)
+
+    expect(presenter.completed_grant_payment_certificate_received).to eql(tasks_data.receive_grant_payment_certificate_date_received)
```

Comment:
Can you change the expectation so that it expect a human-readable date, i.e. 1 1 2024 (or whatever the govuk date format produces!)